### PR TITLE
[luci-interpreter] Enable CircleSin

### PR DESCRIPTION
**Pull request by seanshpark**
This will enable execution of CircleSin Op.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #11760 , from #11778

File: compiler/luci-interpreter/src/kernels/Sin.test.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  Shape input_shape{1, 1, 3};
+  std::vector<int64_t> input_data{1l, 2l, 3l};
+  Tensor input_tensor =
+    makeInputTensor<DataType::S64>(input_shape, input_data, memory_manager.get());
+  Tensor output_tensor = makeOutputTensor(DataType::S64);
+
+  Sin kernel(&input_tensor, &output_tensor);
+  EXPECT_ANY_THROW(kernel.configure());
+}
+
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/Sin.test.cpp:67]: It would be better to undefine PI.

**Review by jinevening — APPROVED** [review]
LGTM with minor comment.

File: compiler/luci-interpreter/src/kernels/Sin.test.cpp (hunk 1 above)
Comment (seanshpark) [reply compiler/luci-interpreter/src/kernels/Sin.test.cpp:67]: May I ask the reason?

**Review by BalyshevArtem — APPROVED** [review]
LGTM

File: compiler/luci-interpreter/src/kernels/Sin.test.cpp (hunk 1 above)
Comment (jinevening) [reply compiler/luci-interpreter/src/kernels/Sin.test.cpp:67]: Ah, sorry for confusion. I misunderstood. It's defined in .cpp, so undef is unnecessary.

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**